api/src/main/java/io/druid/indexer/TaskStatusPlus.java

```diff
@@ -44,7 +45,8 @@ public TaskStatusPlus(
       @JsonProperty("queueInsertionTime") DateTime queueInsertionTime,
       @JsonProperty("state") @Nullable TaskState state,
       @JsonProperty("duration") @Nullable Long duration,
-      @JsonProperty("location") TaskLocation location
+      @JsonProperty("location") TaskLocation location,
+      @JsonProperty("dataSource") String dataSource
```

Comment:
I don't think an indexing service task must have a datasource associated with it. Is there a reason it must be so here?

Reply:
As per my understanding the datasource should be associated with it. You can suggest me if its not true.

Reply:
you are correct as per the comments on `io.druid.indexing.common.task.Task#getDataSource`